helm/o11y-canary/values.yaml

```
@@ -0,0 +1,88 @@
+replicaCount: 1
+image:
+  repository: ghcr.io/linode-obs/o11y-canary
+  tag: v0.1.0
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port: 8080
+
+# Single cert secret for all endpoints
+# TODO - expand this?
+# You must create this secret yourself!
+# The secret must contain keys: ca.crt, tls.crt, tls.key
+certSecretName: o11y-canary-tls
+vaultCertIssuer: vault-production-o11y-canary
+certificate:
+  commonName: o11y-canary.example.com
+
+canaryConfig: |-
+  canary:
+    my_canary_1:
+      type: metrics
+      tls:
+        enabled: true
+        ca_file: /certs/ca.crt
+        cert_file: /certs/tls.crt
+        key_file: /certs/tls.key
+        server_name: otel-collector
+      ingest:
+        - url: otel-collector:4317
+      query:
+        - url: http://vm-singleton:8428
+          tls:
+            enabled: false
+      additional_labels:
+        environment: staging
+      interval: 5s
+      write_timeout: 10s
+      query_timeout: 60s
+      max_active_canaried_series: 5
```

```suggestion
canaryConfig:
  canary:
    my_canary_1:
      type: metrics
      tls:
        enabled: true
        ca_file: /certs/ca.crt
        cert_file: /certs/tls.crt
        key_file: /certs/tls.key
        server_name: otel-collector
      ingest:
        - url: otel-collector:4317
      query:
        - url: http://vm-singleton:8428
          tls:
            enabled: false
      additional_labels:
        environment: staging
      interval: 5s
      write_timeout: 10s
      query_timeout: 60s
      max_active_canaried_series: 5
```
I _think_ you should be able to make this a yaml object instead of a multi-line string here and still just template it into a multi-line string in the configmap template. That'll give us proper yaml syntax highlighting in the values file for the config